app/services/pairing_strategies/single_sided_swiss.rb

```diff
@@ -94,19 +98,15 @@ def self.preferred_player1_side(player1_side_bias, player2_side_bias)
 
     def assign_byes!
```

Comment:
It's very difficult to tell what's different between the two implementations of these methods now, especially as both are changed. Is it possible to move the shared ones to one place?